_posts/2018-06-29-14-04-netcore-7-0.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title : "14.4 OData Web API 7.0"
+description: "7.0 release notes"
+category: "14. 7.x Features "
+---
+
+We're happy to announce the release of ASP.NET Web API OData 7.0 on the [NuGet gallery](https://www.nuget.org/)!
+
+ASP.NET Web API OData 7.0 is available with two new packages:
+
+1. [Microsoft.AspNetCore.OData](https://www.nuget.org/packages/Microsoft.AspNetCore.OData) is based on ASP.NET Core 2.0.
+2. [Microsoft.AspNet.OData](https://www.nuget.org/packages/Microsoft.AspNet.OData) is based on ASP.NET Web API.
+
+[Here](http://odata.github.io/WebApi/#14-01-netcore-beta1) can find beta1, beta2 & beta4 information.
```

Comment:
The title is "7.0 Beta1 & Beta2 & Beta4", but this also references 7.0 RTM.  Can we remove references to the individual beta and update this text to be something like "Information on WebAPI OData 7.0 is available [here].  Ideally we would also update section 14.1 to describe the 7.0 release, and remove beta-specific info.

Reply:
changed.